apps/crn-server/src/data-providers/contentful/reminder.data-provider.ts

```diff
@@ -247,14 +247,14 @@ export const getResearchOutputFilter = (
           { createdDate_gte: last24HoursISO },
           { sys: { publishedVersion_exists: false } },
           { addedDate_exists: false },
-          { reviewRequestedBy_exists: false },
+          { statusChangedBy_exists: false },
         ],
       },
       {
         AND: [
           { sys: { publishedVersion_exists: false } },
           { addedDate_exists: false },
-          { reviewRequestedBy_exists: true },
+          { statusChangedBy_exists: true },
```

Comment:
Hey, I believe the condition is wrong here. It should be the same as the change made in squidex

<img width="1098" alt="Screenshot 2023-07-20 at 06 39 41" src="https://github.com/yldio/asap-hub/assets/16595804/7329f0ac-dea7-4f18-85d0-a2999a097e0a">

so here we should also check if `isInReview` is true and above if it is false